### DISCUSSION TITLE
Some osu!mania maps are still way too strict on HP

### DISCUSSION THREAD

**Discussion by Firmatorenio** [ppy/osu#16295] [Q&A] [answered]
While in the majorty of the mania maps the HP drain has been fixed, it would seem there are still cases of it being unreasonably strict. I was directed in #10566 to create a discussion to track that issue.

In my case, the drain of [this map (7k Hard difficulty)](https://osu.ppy.sh/beatmapsets/460142#mania/1002357) feels unnecessarily harsh - you still lose HP even if you hit no GOODs or below and you have to hit a lot of PERFECTs in order to gain health, resulting in fails on the map that feel very cheap, even for a map short as this.

In [this](https://www.youtube.com/watch?v=cWCeLDdtsMw) video, in the section between 0:31 and 0:38 there was no hits of GOOD or worse (as seen on the colour hit meter), yet there's no HP gain to be seen and that causes a fail on the first minor screwup. And even autoplay struggles to keep the HP at max for the majority of the map, making me think that hitting GREAT gives way too much less health compared to PERFECT.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/16429
